01-backend.yaml

```diff
@@ -49,20 +100,20 @@ spec:
         - -config.file=/etc/loki/loki.yaml
         image: grafana/loki:2.0.0
         imagePullPolicy: IfNotPresent
-        livenessProbe:
-          httpGet:
-            path: /ready
-            port: http-metrics
-          initialDelaySeconds: 45
+#        livenessProbe:
```

Comment:
Why did you comment the Probes?

Reply:
I left Liveness and Readiness disabled, to simplify use in laboratories.
Since the main objective is to quickly learn or test concepts, the additional complexity of the probes is not necessary.
By disabling probes, containers can be deployed faster because there is no need to wait for probe-specified health checks.